iocage/lib/ioc_start.py

```diff
@@ -332,36 +333,31 @@ def start_network_interface_vnet(self, nic, net_configs, jid):
                             err = "\n  Invalid interface supplied: {}"
                             self.lgr.error(err.format(iface))
                             self.lgr.error("  Did you mean {}?\n".format(nic))
-                        else:
-                            self.start_network_vnet(nic, bridge, membermtu,
-                                                    iface, ip, gw, jid)
+                            continue
+                        if iface not in ifaces:
+                            self.start_network_vnet_iface(nic, bridge,membermtu,
+                                                         iface, ip, gw, jid)
+                            ifaces.append(iface)
+
+                        self.start_network_vnet_addr(nic, bridge,membermtu,
+                                                     iface, ip, gw, jid)
 
         except CalledProcessError as err:
             self.lgr.warning("Network failed to start:"
                              f" {err.output.decode('utf-8')}".rstrip())
 
-    def start_network_vnet(self, nic, bridge, mtu, iface, ip, defaultgw, jid):
+    def start_network_vnet_iface(self, nic, bridge, mtu, iface, ip, defaultgw, jid):
```

Comment:
Some variables are no longer used, they should be removed from the definition and all calls.